src/mobile/src/components/Banner/BannerText.jsx

```diff
@@ -25,19 +25,38 @@ const styles = StyleSheet.create({
   },
 });
 
+async function fetchStudentQuotes() {
+  const { data, error } = await supabase.from('quotes').select('*');
+
+  if (error) {
+    throw new Error(error.message);
+  }
+
+  return data;
+}
+
 const BannerText = () => {
-  const [studentQuote, setStudentQuote] = useState(quotes[0]);
+  const [studentQuotes, setStudentQuotes] = useState({});
 
   useEffect(() => {
-    setStudentQuote(quotes[Math.floor(Math.random() * quotes.length)]);
+    fetchStudentQuotes()
+      .then((quotes) => {
+        const randomIndex = Math.floor(Math.random() * quotes.length);
+        setStudentQuotes(quotes[randomIndex]);
+        return quotes;
```

Comment:
why `return quotes`?

Reply:
When I try to write `then` they ask to have some value returned

Reply:
I think for our `eslint` config we have to either return and throw for all flows or not return or throw at all

Reply:
Why don't we use `await`?